components/git/security.js

```diff
@@ -26,6 +26,10 @@ const securityOptions = {
   'pre-release': {
     describe: 'Create the pre-release announcement',
     type: 'boolean'
+  },
+  'pos-release': {
```

Comment:
```suggestion
  'post-release': {
```

as Richard mentioned in the channel